content/en/real_user_monitoring/guide/devtools-tips.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Tips when using browser's developer tools
+kind: guide
+further_reading:
+    - link: '/real_user_monitoring/explorer'
+      tag: 'Documentation'
+      text: 'Visualize your RUM data in the Explorer'
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏no idea if there is some rules for that but suggesting explorer doc here seems odd to me.
I would say maybe having the general browser RUM doc and the general browser log doc could be nice.

Reply:
👌